wp-content/themes/access/views/single-page.twig

```diff
@@ -54,8 +57,8 @@
     </div>
 
     <footer class="usa-grid py-4">
-      <p class="text-color-grey-mid m-0">
-        {{ __('Updated', 'accessnyc-page') }} {{ post.post_date_modified|date }}
+      <p class="text-color-grey-mid m-0" itemtype="https://schema.org/dateModified">
```

Comment:
`itemtype`s should be `itemprop` within the scope of the `WebPage` schema and you can drop the url.

```
itemprop="dateModified">
```